src/utilities/url-query.ts

```diff
@@ -0,0 +1,16 @@
+import queryString from "query-string";
+
+/**
+ * Simplifies query-string library by only returning `string | undefined`, instead
+ * of `string | string[] | null | undefined`.
+ * @param prop
+ */
+export const queryValue = (prop: string): string | undefined => {
+  const query = queryString.parse(window.location.search);
+  const val = query[prop];
+  if (!val) return;
```

Comment:
It's a distinction that I've never liked much personally (for example, I much prefer `?student=true` to `?student`), but in any case, this was deliberately made to handle the minimal requirements so far, rather than guess how we'll use parameters in the future. E.g. there is no casting of booleans or numbers yet either.

Reply:
Agreed that it's not an immediate concern. My feeling on boolean parameters is that it's an unnecessary cognitive burden on the "user" to have to remember whether it should be `true` or `yes` or `1` or something else. I've worked on apps that ended up with monstrosities like `?bool1=true&bool2=1&bool3=yes&bool4`, which may have colored my thinking to some extent. 😉